test/verify/machineslib.py

```diff
@@ -1298,6 +1298,61 @@ def installWithErrorTest(dialog):
                                          storage_pool="No Storage",
                                          start_vm=True,))
 
+        # Test create VM with disk of type "block"
+        self.machine.add_disk("50M", serial="DISK1")
+        wait(lambda: self.machine.execute("ls -l /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_DISK1"))
+        cmds = [
+            "parted /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_DISK1 mklabel gpt",
+            "virsh pool-define-as poolDisk disk - - /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_DISK1 - /tmp/poolDiskImages",
+            "virsh pool-build poolDisk --overwrite",
+            "virsh pool-start poolDisk",
+            "virsh vol-create-as poolDisk sda1 1024"
+        ]
+        self.machine.execute(" && ".join(cmds))
+
+        self.browser.reload()
+        self.browser.enter_page('/machines')
+        self.browser.wait_in_text("body", "Virtual Machines")
+
+        # Check choosing existing volume as destination storage
+        createThenInstallTest(TestMachines.VmDialog(self, sourceType='file',
+                                         location=config.NOVELL_MOCKUP_ISO_PATH,
+                                         memory_size=50, memory_size_unit='MiB',
+                                         storage_pool="poolDisk",
+                                         storage_volume="sda1"))
+
+        if "debian" not in self.machine.image and "ubuntu" not in self.machine.image:
+            # Test create VM with disk of type "network"
+            target_iqn = "iqn.2019-09.cockpit.lan"
+            orig_iqn = self.machine.execute("sed </etc/iscsi/initiatorname.iscsi -e 's/^.*=//'").rstrip()
+
+            # Increase the iSCSI timeouts for heavy load during our testing
+            self.machine.execute("""sed -i 's|^\(node\..*log.*_timeout = \).*|\\1 60|' /etc/iscsi/iscsid.conf""")
+
+            # Setup a iSCSI target
```

Comment:
This code is duplicated, I use it in check-machines-dbus as well. Do you mind to extract some helper method (createISCSIPool) and re-use it?